src/app/services/auth/auth.service.ts

```diff
@@ -38,4 +41,11 @@ constructor(private router: Router) { }
     this.router.navigate(['/']);
   }
 
+  getUserById(id:number):Observable<User>{
+    return this.http.get<User>(`${environment.userUri}/${id}`)
+  }
```

Comment:
I believe we're going to be moving this to a user service, so we can delete it from here (unless its currently needed)